culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+> If you want to go fast, go alone. If you want to go far, go together.
+
+As a community, we value going far over going fast. To go far, we need to stay
+aligned in our values, goals, and practices. One of the ways that we do this is
+through cultivating review culture.
+
+# On Review Culture
+
+Review culture came to us from the practices of software engineers. Large
+codebases are incredibly complex: they contain the ideas and work of hundreds
+of people, manifested in millions of lines of code. They need to remain
+logically consistent, even as they grow and change. A big part of how engineers
+manage is through code review: every single change to the system is reviewed by
+at least one other person before becoming official, or "merging".
+
+Ensuring that all work is reviewed brings several benefits. Most obviously, it
+leads to higher quality work, because the final output includes feedback and
+suggestions from others. Just as importantly, reviews enable the exchange of
+knowledge, context, and perspective within the team. In a good review, both the
+contributor and the reviewer can teach each other about how work gets done on
+the project. Finally, review ensures that every part of the project is
+understood by at least two people.
+
+# The Review Process
+
+## Changes
+The review process orients around _changes_, which are the units of work being
+reviewed.
+
+A change can take many forms, for example:
+- Adding a new guide or piece of documentation
+- Adding a new module to the codebase
+- Modifying an existing guide or module
+
+
+## Participants in the Review Process
+
+### Contributor
+
+The contributor is the person proposing a change. Anyone can decide to be a
+contributor: the only requirements are that you have a change you need to
+propose, and that you agree to be a respectful participant in the review
+process. (When we have a code of conduct--link to it here.)
+
+### Reviewer
+
+The reviewers are the people who review the change, suggest changes to it, and
+eventually approve it. Anyone in the community can act as a reviewer: the only
+requirements are that you have some insight to share (or questions to ask)
```

Comment:
There is a subtle detail here - anyone in the community can act as a reviewer via commenting, but only organization members can appear next to the PR in the little list to act as "official" reviewers. Are we making a distinction?

Reply:
I didn't realize you need permissions to add a review in GitHub. This distinction can matter for cred tracking purposes (since a review node is distinct from a comment node). However, for this gude, I'm trying to describe review practices as abstracted from any particular platform. Right now we're mostly doing reviews on GitHub, but in the future we may have a 1st party system for reviewing cred changes, etc.

Reply:
> I didn't realize you need permissions to add a review in Github.

You _can_ add a "review" when not part of the Github organization (I think @s-ben wasn't added yet for example). However in the reviews show up in grey instead of color, and the "approve" and "request changes" option are not made available. You can only use the comment type of review.

I believe the rationale here is that an external contributor can't progress or block a PR.

A screenshot of this from my unprivileged account.

![image](https://user-images.githubusercontent.com/497556/72725338-c9c45d80-3b85-11ea-8665-3913331fa019.png)